lib/base/Version.js

```diff
@@ -183,4 +183,70 @@ Version.prototype.readLimits = function(opts) {
   };
 };
 
+Version.prototype.isCallbackFunction = function(operationPromise, callback) {
+  if (typeof callback === 'function') {
+    operationPromise = operationPromise
+      .then(value => callback(null, value))
+      .catch(error => callback(error));
+  }
```

Comment:
`is...` functions generally just test something and return a boolean. The name should reflect the actions taken.